ragna/deploy/_ui/components/file_uploader.py

```diff
@@ -101,10 +105,6 @@ def perform_upload(self, event=None, after_upload_callback=None):
                 .fileUploadDropArea.draggedOver {
                     border-width: 3px;
                 }
-
```

Comment:
Removed to reduce the number of resize operations on the height of the upload container (see previous comment on how I have introduced a new parameter for this). This has negligible UX impact IMO. The basic functionality was to make the upload container smaller when a 'pill' (icon for a newly upload file) was present below. I have now introduced some more spacing and a scroll (see below).